tests/unit/util/test.inc

```diff
@@ -177,9 +177,9 @@ void test_UTIL_LISTIFY(void)
 	UTIL_LISTIFY(4, DEFINE, 2)
 	UTIL_LISTIFY(4, MARK_UNUSED, _)
 
-	UTIL_LISTIFY(4, INC, _)
+	UTIL_LISTIFY(4, INC, 1, 2)
 
-	zassert_equal(i, 0 + 1 + 2 + 3, NULL);
+	zassert_equal(i, 0 + 1 + 2 + 3 + 4 * (1 + 2), NULL);
```

Comment:
This would be a little more clear expressed as:

    zassert_equal(i, (0 + 1 + 2) + (1 + 1 + 2) + (2 + 1 + 2) + (3 + 1 + 2), NULL);

since that represents the effect of the four calls.